framework/src/iotbus/iotbus_common.c

```diff
@@ -0,0 +1,85 @@
+/****************************************************************************
+ *
+ * Copyright 2019 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+#include <iotbus/iotbus_common.h>
+
+#include <sys/types.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <signal.h>
+#include <errno.h>
+#include <pthread.h>
+
+#define INTR_THREAD_STACK_SIZE		2048
+
+char *iotbus_pin_str[IOTBUS_PIN_MAX] = {
```

Comment:
This definition had better moved to another place where IOT_BUS_PIN_XX defined. It will help you maintain more easily.